ruby/engine/InitRubyEngine.cpp

```diff
@@ -14,6 +14,9 @@ extern "C"
 {
   // static void Init_builtin_prelude(void);
   // void Init_builtin_gem_prelude(void);
+  struct ruby_cmdline_options;
+  typedef struct ruby_cmdline_options ruby_cmdline_options_t;
+  void Init_ruby_description(ruby_cmdline_options_t*);
```

Comment:
Forward declare.

In 2.7.2, it was simpler, just `void Init_ruby_description(void)`